nxlog/package.yaml

```diff
@@ -0,0 +1,86 @@
+id: nxlog
+name: NXLog
+author: Tenzir
+author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
+package_icon: https://raw.githubusercontent.com/tenzir/library/main/nxlog/package.svg
+
+description: |
+
+
+inputs:
+  url:
+    name: URL
+    description: |
+      A URL under which tenzir shall receive data from NXlog.
+
+      To send logs straight to a TCP socket, use the [TCP output
+      module](https://docs.nxlog.co/agent/current/om/tcp.html) with the following
+      configuration:
+
+      ```
+      <Extension json>
+        Module    xm_json
+      </Extension>
+
+      <Output tcp>
+        Module    om_tcp
+        Host      10.0.0.1:1514
+        Exec      to_json();
+      </Output>
+      ```
+    type: string
+    default: tcp://0.0.0.0:1514
+
+  tls:
+    name: TLS
+    description: |
+      Use transport level security to encrypt data in transit.
+      Use the [SSL output
+      module](https://docs.nxlog.co/agent/current/om/ssl.html) instead of
+      om_tcp.
+
+      Example nxlog.conf:
+      ```
+      <Extension json>
+        Module    xm_json
+      </Extension>
+
+      <Output ssl>
+        Module          om_ssl
+        Host            10.0.0.1:1514
+        CAFile          %CERTDIR%/ca.pem
+        CertFile        %CERTDIR%/client-cert.pem
+        CertKeyFile     %CERTDIR%/client-key.pem
+        KeyPass         secret
+        AllowUntrusted  TRUE
+        OutputType      Binary
+        Exec            to_json();
+      </Output>
+    type: boolean
+    default: "false"
```

Comment:
I haven't tested whether this works as expected yet. I assume that quoting is necessary here because the backend currently treats all inputs as strings.

Reply:
It won't be necessary, we (for historical reasons) do a print-parse cycle here.